tests/refc/memory-leak/run

```diff
@@ -0,0 +1,4 @@
+rm -rf build
+
+$1 --no-banner --no-color --console-width 0 --cg refc -o main Main.idr > /dev/null
+valgrind -q --leak-check=full ./build/exec/main
```

Comment:
Would the `$VALGRIND` env var suffice here or is the use-case different enough that it wouldn't apply with the same flags? I know the VALGRIND env var was a recent addition that you've only merged into your branch just now.